packages/xstate-react/src/createActorContext.tsx

```diff
@@ -46,24 +42,25 @@ export function createActorContext<TMachine extends AnyStateMachine>(
     }
 
     return actor;
-  };
-
-  ReactContext.useActor = () => {
-    const actor = useContext(ReactContext) as any;
+  }
 
-    return useActor(actor);
-  };
+  function useActor() {
+    const actor = useContext();
+    return useActorUnbound(actor);
+  }
 
-  ReactContext.useSelector = <T,>(
+  function useSelector<T>(
     selector: (snapshot: EmittedFrom<TMachine>) => T,
     compare?: (a: T, b: T) => boolean
-  ): T => {
-    const actor = useContext(ReactContext) as any;
```

Comment:
Fixed the usage of `useContext` here. We should grab it from the "validated" `useContext` and not directly from the created React context - this way we will throw if the user forgets to use the provider.